easy_access/db/update.py

```
@@ -1635,28 +1773,54 @@ async def update_workflow_status_from_db(settings: Settings) -> None:
+        )
+        file_exists_count = result1.rowcount
+
+        # Perform bulk update for items with done manual classifications
+        result2 = await session.execute(
+            text(
+                f"""
+                UPDATE copyright_data
+                SET workflow_status = 'Done'
+                WHERE workflow_status != 'Done'
+                  AND LOWER(manual_classification) IN ({done_classifications})
+                """
```

Ruff (0.14.6)

Possible SQL injection vector through string-based query construction

(S608)